### ISSUE TITLE
Any chance of getting this boosted to latest? (v6.2?) 

### ISSUE THREAD

**Issue by thestraycat** [paulgalow/fileshelter-docker#5]
As above really. I really like your container as it plays nice with remote share links where as the official repo dosn't seem to play ball :(

**Comment by paulgalow** [comment]
Thanks for writing in. I'm afraid I'm no longer using it, hence the lack of activity. And it seems a lot has changed since then.

**Comment by thestraycat** [comment]
No worries i thought that might be the case :)  

Did you move on to using something else? Or do you simply not self host anything to share files anymore?

**Comment by paulgalow** [comment]
Right, I'm still searching for a great solution that takes less maintenance on my end but can still be considered more safe than simply uploading something on a public hosting service. [Tresorit Send](https://send.tresorit.com/) seems promising in that regard, but I haven't dug deep.
